tests/utils.py

```diff
@@ -92,7 +92,7 @@ def sampleEgoActions(
         asMapping=False,
         timestep=timestep,
     )
-    return [actions[0] for actions in allActions]
+    return [actions[0] if actions else None for actions in allActions]
```

Comment:
What is this change for? Shouldn't we always return whatever action(s) came out of `sampleActions` here? (The latter already converts an empty tuple to `None` if `singleAction` is enabled.)

Reply:
Hmm, I added this because at this stage actions was either empty or `None` so `actions[0]` was causing an exception.